.travis.yml

```diff
@@ -27,6 +27,8 @@ before_script:
   - export MVI_URL='http://ps-dev.commserv.healthevet.va.gov:8110/psim_webservice/IdMWebService'
   - export VHA_MAPSERVER_URL='https://maps.va.gov/server/rest/services/PROJECTS/Facility_Locator/MapServer'
   - export VHA_MAPSERVER_LAYER='0'
+  - export NCA_MAPSERVER_URL='https://services3.arcgis.com/aqgBd3l68G8hEFFE/ArcGIS/rest/services/NCA_Facilities/FeatureServer'
```

Comment:
is "aqgBd3l68G8hEFFE" an app token? Just want to make sure in the event that this is getting checked into source control that this is fake, and not real app token.